generators/entity-server/templates/src/main/java/package/service/NoDbService.java.ejs

```diff
@@ -17,36 +17,44 @@
  limitations under the License.
 -%>
 package <%= packageName %>.service;
+<%_
+const instanceType = restClass;
+const instanceName = restInstance;
+%>
 
+<% if (dtoMapstruct) { %>
+import <%= packageName %>.service.dto.<%= dtoClass %>;
+<%_ } else { _%>
 import <%= packageName %>.domain.<%= asEntity(entityClass) %>;
+<%_ } _%>
 
 import java.util.Optional;
-<%_ if (pagination === 'no') { _%>
+<% if (pagination === 'no') { %>
```

Comment:
I think we can keep the _ here to remove the blank line. But not a big deal if we keep it like that.
```suggestion
<%_ if (pagination === 'no') { _%>
```